src/Geosuggest.tsx

```diff
@@ -615,6 +626,7 @@ export default class extends React.Component<IProps, IState> {
         onSuggestMouseOut={this.onSuggestMouseOut}
         onSuggestSelect={this.selectSuggest}
         renderSuggestItem={this.props.renderSuggestItem}
+        listId={this.listId}
```

Comment:
Shall we pass `id` down instead of `listId` since it’s the `SuggestList` component?

Reply:
we could 🙂